.github/workflows/deploy-to-dev.yaml

```diff
@@ -0,0 +1,97 @@
+name: Dev
+
+on:
+  push:
+    branches:
+      - dev
```

Comment:
Tän vois kokeilla tässä `migrate-dev-to-google-cloud` branchissa viel et toimii lopuksi

Reply:
Kokeilin dev-branchissa. Pelitti kyllä. Katon viel ton Nginx-filun parametrisoinnin